app/src/main/res/values-es/strings.xml

```diff
@@ -235,64 +275,74 @@
   <string name="listallchannels_share_all">Compartir todos los canales…</string>
   <string name="listallchannels_share">Compartir</string>
   <string name="listallchannels_funding_tx">Transacción</string>
-  <string name="listallchannels_funding_tx_desc">Ver transacción de pagos</string>
+  <string name="listallchannels_funding_tx_desc">Ver transacción de financiación</string>
   <string name="listallchannels_close">Cerrar</string>
   <string name="listallchannels_share_subject">Datos de canales</string>
   <string name="listallchannels_share_title">Compartir datos del canal</string>
-  <string name="listallchannels_serialization_error">No se pudieron recuperar los datos del canal.</string>
+  <string name="listallchannels_serialization_error">No se han podido recuperar los datos del canal</string>
 
   <!-- //////////////// channel details //////////////// -->
 
   <string name="channel_loading">Recuperando datos del canal…</string>
-  <string name="channel_error">No se pudieron recuperar los detalles de este canal. Por favor, inténtalo de nuevo más tarde.</string>
+  <string name="channel_error">No se han podido recuperar los detalles de este canal. Por favor, inténtalo de nuevo más tarde.</string>
 
   <!-- //////////////// menu //////////////// -->
 
-  <string name="menu_settings">Configuración</string>
-  <string name="menu_send">Enviar</string>
-  <string name="menu_receive">Recibir</string>
+  <string name="menu_settings">Settings</string>
+  <string name="menu_send">Send</string>
+  <string name="menu_receive">Receive</string>
```

Comment:
Oops!

Please revert this.

Reply:
I used the original english file, that's the reason for this and other I founded, thanks for catch this!